src/Tag/Base/ContentTag.php

```diff
@@ -67,13 +71,16 @@ final public function content(string $content): self
      */
     final protected function generateContent(): string
     {
-        $content = $this->content;
-        if ($this->encode) {
-            $content = Html::encode($this->content, $this->doubleEncode);
-        }
-        if ($this->encodeSpaces) {
-            $content = str_replace(' ', '&nbsp;', $content);
-        }
-        return $content;
+        $content = array_map(function ($item) {
```

Comment:
I'd measure performance of regular `foreach`-based implementation. That's central part of the package that is used a lot.

Reply:
Create bench: https://github.com/yiisoft/html/pull/60

Results:

```
λ vendor\bin\phpbench.bat run tests\ContentBenchmark.php --report=aggregate
PHPBench @git_tag@ running benchmarks...
with configuration file: F:\open-source\yiisoft\html/phpbench.json
with PHP version 8.0.0, xdebug ❌, opcache ❌

\Yiisoft\Html\Tests\ContentBenchmark

    benchArrayMap...........................I9 2,223.102 μs (±1.31%)
    benchForeach............................I9 2,147.450 μs (±1.65%)
    benchWithoutArray.......................I9 2,077.452 μs (±1.18%)

⅀T: 64,813.000μs μSD/r 29.890μs μRSD/r: 1.382%

Subjects: 3, Assertions: 0, Failures: 0, Errors: 0
suite: 134628a70dd90b1446e602b1ff341f546678f1c6, date: 2021-03-14, stime: 10:59:20
+------------------+-------------------+-----+------+-----+------------+-------------+-------------+-------------+-------------+----------+--------+-------+
| benchmark        | subject           | set | revs | its | mem_peak   | best        | mean        | mode        | worst       | stdev    | rstdev | diff  |
+------------------+-------------------+-----+------+-----+------------+-------------+-------------+-------------+-------------+----------+--------+-------+
| ContentBenchmark | benchArrayMap     | 0   | 1    | 10  | 4,468,768b | 2,183.000μs | 2,228.500μs | 2,223.102μs | 2,277.000μs | 29.248μs | 1.31%  | 1.07x |
| ContentBenchmark | benchForeach      | 0   | 1    | 10  | 4,468,184b | 2,127.000μs | 2,165.900μs | 2,147.450μs | 2,237.000μs | 35.717μs | 1.65%  | 1.04x |
| ContentBenchmark | benchWithoutArray | 0   | 1    | 10  | 4,466,128b | 2,060.000μs | 2,086.900μs | 2,077.452μs | 2,151.000μs | 24.704μs | 1.18%  | 1.00x |
+------------------+-------------------+-----+------+-----+------------+-------------+-------------+-------------+-------------+----------+--------+-------+
```

Foreach is faster, but without array is more faster :)

Reply:
Let's use foreach one.